packages/vscode-extension/src/webview/views/LaunchConfigurationView.tsx

```diff
@@ -0,0 +1,267 @@
+        ref={productFlavorInputRef}
+        className="input-configuration"
+        type="string"
+        defaultValue={productFlavor ?? "Auto"}
+        onBlur={onProductFlavorBlur}
+      />
+    </div>
+  );
+}
+
+interface appRootConfigurationProps {
+  appRoot?: string;
+  update: <K extends keyof LaunchConfigurationOptions>(
+    key: K,
+    value: LaunchConfigurationOptions[K]
+  ) => void;
+}
+
+function AppRootConfiguration({ appRoot, update }: appRootConfigurationProps) {
+  const appRootInputRef = useRef<HTMLInputElement>(null);
+
+  const onAppRootBlur = () => {
+    const newAppRoot = appRootInputRef.current?.value;
+    if (newAppRoot === "Auto" || !newAppRoot) {
+      update("appRoot", "Auto");
+      return;
+    }
+    update("appRoot", newAppRoot);
+  };
+
+  return (
+    <div className="container">
+      <div className="setting-description">App Root:</div>
+      <input
+        ref={appRootInputRef}
+        className="input-configuration"
+        type="string"
+        defaultValue={!!appRoot ? appRoot : "Auto"}
+        onBlur={onAppRootBlur}
+      />
+    </div>
+  );
+}
+
+interface metroPathConfigurationProps {
+  metroConfigPath?: string;
+  update: <K extends keyof LaunchConfigurationOptions>(
+    key: K,
+    value: LaunchConfigurationOptions[K]
+  ) => void;
+}
+
+function MetroConfigPathConfiguration({ metroConfigPath, update }: metroPathConfigurationProps) {
+  const metroPathInputRef = useRef<HTMLInputElement>(null);
+
+  const onMetroPathBlur = () => {
+    const newMetroPath = metroPathInputRef.current?.value;
+    if (newMetroPath === "Auto" || !newMetroPath) {
+      update("metroConfigPath", "Auto");
+      return;
+    }
+    update("metroConfigPath", newMetroPath);
+  };
+
+  return (
+    <div className="container">
+      <div className="setting-description">Metro Config Path:</div>
+      <input
+        ref={metroPathInputRef}
+        className="input-configuration"
+        type="string"
+        defaultValue={!!metroConfigPath ? metroConfigPath : "Auto"}
+        onBlur={onMetroPathBlur}
+      />
+    </div>
+  );
+}
+
+interface isExpoConfigurationProps {
+  isExpo?: boolean;
+  update: <K extends keyof LaunchConfigurationOptions>(
+    key: K,
+    value: LaunchConfigurationOptions[K]
+  ) => void;
+}
+
+function IsExpoConfiguration({ isExpo, update }: isExpoConfigurationProps) {
+  const options = [
+    { value: "true", label: "Yes" },
+    { value: "false", label: "No" },
+    { value: "Auto", label: "Auto" },
+  ];
+
+  const onIsExpoChange = (newIsExpo: string) => {
+    if (newIsExpo === "true") {
+      update("isExpo", true);
+      return;
+    } else if (newIsExpo === "false") {
+      update("isExpo", false);
+      return;
+    }
+    // @ts-ignore
+    update("isExpo", "Auto");
+  };
+
+  return (
+    <div className="container">
+      <div className="setting-description">Is Expo:</div>
+      <Select
+        value={isExpo !== undefined ? isExpo.toString() : "Auto"}
```

Comment:
`isExpo?.toString() ?? "Auto"`

Reply:
unfortunately no because isExpo can be false and false is falsy